src/rgw/setup/rgw.py

```diff
@@ -274,7 +274,7 @@ def _create_rgw_user(conf: MappedConf):
         rgw_config = Rgw._get_rgw_config_path(conf)
         create_usr_cmd = f'radosgw-admin user create --uid={user_name} --access-key \
             {access_key} --secret {password} --display-name="{user_name}" \
-            --caps="users=*;metadata=*;usage=*;zone=*" \
+            --caps="users=*;metadata=*;usage=*;zone=*;info=*;user=*;roles=*;user-policy=*;buckets=*" \
```

Comment:
Enabling all permissions blindly can raise some security concerns later, we should add specific roles & capabilities and not wildcard * i believe.
@sachinpunadikar aren't there any specific values we should be adding ?

Reply:
For all capabilities there are only 2 values, READ & WRITE like below:
#define RGW_CAP_READ            0x1
#define RGW_CAP_WRITE           0x2
#define RGW_CAP_ALL             (RGW_CAP_READ | RGW_CAP_WRITE)
So here * means both capabilities. To get the values we need READ & and for set/modify, we need WRITE capability. So current capabilities to admin user are fine, no need to change.

Reply:
Thanks @sachinpunadikar for confirming. @shriya-deshmukh old me on simlar lines yday, still wanted to double confirm